lib/database/__init__.py

```diff
@@ -271,7 +271,7 @@ def find(self, q, operator="=", fields="*", key=None):
         
         if key == None: key = self._key
         if fields != "*": fields = ", ".join(fields)
-        try: q = unicode(q)
+        try: q = str(q)
         except: pass
```

Comment:
I'm not sure about the wisdom of this exception handling, should we just do `q = str(q)` and be done with it ?